libraries/AP_HAL_ChibiOS/hwdef/FlywooH743Pro/README.md

```diff
@@ -0,0 +1,100 @@
+# Flywoo H743 Pro Flight Controller
+
+The Flywoo H743 Pro is a flight controller produced by [Flywoo](https://www.flywoo.net/).
+
+## Features
+
+ - MCU - STM32H743 32-bit processor running at 480 MHz
+ - IMU - Dual ICM42688
+ - Barometer - SPL06
+ - OSD - AT7456E
+ - Onboard Flash: 500MByte
+ - 7x UARTs
+ - 13x PWM Outputs (12 Motor Output, 1 LED)
+ - Battery input voltage: 2S-6S
+ - BEC 3.3V 0.5A
+ - BEC 5V 3A
+ - BEC 10V 3A
+
+## Pinout
+
+![Flywoo H743 Pro Board Top](Top.png "Flywoo H743 Pro Top")
+![Flywoo H743 Pro Board Bottom](Bottom.png "Flywoo H743 Pro Bottom")
+
+## UART Mapping
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+
+ - SERIAL0 -> USB
+ - SERIAL1 -> UART1 (RX/SBUS, DMA-enabled)
+ - SERIAL2 -> UART2 (GPS, DMA-enabled)
+ - SERIAL3 -> UART3
+ - SERIAL4 -> UART4 (GPS, DMA-enabled)
+ - SERIAL6 -> UART6 (ESC Telemetry)
+ - SERIAL7 -> UART7
+ - SERIAL8 -> UART8 (DJI/MSP, DMA-enabled)
+
+## RC Input
```

Comment:
replace RC section with this:
RC input is configured by default via the USART1 RX input. It supports all unidirectional RC protocols except PPM. 

Note: If the receiver is FPort the receiver must be tied to the USART1 TX pin ,  RSSI_TYPE set to 3, and SERIAL2_OPTIONS must be set to 7 (invert TX/RX, half duplex). For full duplex like CRSF/ELRS use both RX1 and TX1 and set RSSI_TYPE also to 3.